### PR TITLE
Text changes

### PR DESCRIPTION
**Pull request by pcuk1979**
Content changes from tickets AS-1487 and AS-1351.

Let me know if this causes any issues with anything.

### CODE REVIEW

File: pages/profile/roles/apply/content/confirm.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ const { merge } = require('lodash');
...
 module.exports = merge({}, baseContent, {
-  reviewRoleApplication: 'Please review your application for a named role',
-  applyingFor: 'You are applying for the role:',
-  onBehalfOf: 'On behalf of:',
+  reviewRoleApplication: 'Assign named role',
+  applyingFor: 'You are assigning the role of:',
+  onBehalfOf: 'To:',
   rcvsNumber: 'RCVS number:',
-  explanation: 'Explain why this person is suitable for the role:',
+  explanation: 'Why is this person suitable for this named role:',
```
Comment (lennym) [inline pages/profile/roles/apply/content/confirm.js]: This probably wants a `?` now.

File: pages/profile/roles/apply/content/index.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ module.exports = merge({}, baseContent, {
   title: 'Amend profile',
   fields: {
     type: {
-      label: 'Please select the role you want to apply for.'
+      label: 'Which named role do you want to assign?.'
```
Comment (lennym) [inline pages/profile/roles/apply/content/index.js]: You have a question mark and full stop here.

Comment (pcuk1979) [reply pages/profile/roles/apply/content/index.js]: Changed

File: pages/profile/roles/apply/content/confirm.js (hunk 1 above)
Comment (pcuk1979) [reply pages/profile/roles/apply/content/confirm.js]: Changed

**Pull request merged**